spec/features/timeout_spec.rb

```diff
@@ -4,8 +4,8 @@
   let(:one_second_in_minutes) { 1 / 60.to_f }
 
   before do
-    allow_any_instance_of(ClaimsHelper).to receive(:claim_timeout_in_minutes) { one_second_in_minutes }
-    allow_any_instance_of(ClaimsHelper).to receive(:claim_timeout_warning_in_minutes) { one_second_in_minutes }
+    allow_any_instance_of(ApplicationController).to receive(:claim_timeout_in_minutes) { one_second_in_minutes }
```

Comment:
It's a shame these feature specs rely so heavily on stubs and setting expectations on method calls as it makes them brittle as demonstrated by the need for these changes. Do you fancy rewriting them to use `travel` to verify the behaviour instead?

Reply:
I seem to remember trying `travel` when writing these originally but they're using JavaScript for the timer for the modal popup thing so it wasn't working.

Reply:
Ah, yeah. I remember this now. We could use Sinion or something to stub the time, but this adds a layer of complication